src/components/App.jsx

```diff
@@ -34,7 +34,7 @@ const ContactsApp = props => {
         {isMobile && (
           <BarCenter>
             <MuiCozyTheme>
-              <Typography variant="h4">Contacts</Typography>
+              <Typography variant="h4">{appName}</Typography>
```

Comment:
I think the appName is already present in cozy-client, and that instead of being threaded through props, it should be accessed via cozy-client.